Task2_StudentsAndTeachers/Classroom.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_StudentsAndTeachers
+{
+    internal class Classroom
+    {
+        public Teacher Teacher { get; set;}
+        public SortedDictionary<int, Student> StudentsDatabase { get; set;}
+        public static List<Student> StudentsIndexed { get; set;}
+        public int StudentCount { get; set;}
+
+
+        public Classroom()
+        {
+            StudentsDatabase = new SortedDictionary<int, Student>();
+            StudentsIndexed = new List<Student>();
+        }
+
+        public void PrintClassroom()
+        {
+            Console.WriteLine("\n ===================== \n");
+            Console.WriteLine(Teacher.ToString());
+            Console.WriteLine("*******");
+            foreach (KeyValuePair<int, Student> entry in StudentsDatabase)
+            {
+                Console.WriteLine(entry.Value.ToString());
+            }
+        }
+        public void AddStudent(Student s)
+        {
+            StudentsDatabase.Add(StudentCount, s);
+            StudentsIndexed.Add(s);
+            StudentCount++;
+        }
+
+        public int GetIdByGuid(Guid guid)
+        {
+            foreach (KeyValuePair<int, Student> pair in StudentsDatabase)
+            {
+                if(pair.Value.GuidIdentifier == guid) return pair.Key;
+                
+            }
+            return 0;
+        }
+
+        public void ChangeGradeById(int id, string grade)
+        {
+            if (StudentsDatabase.TryGetValue(id, out Student s))
+            {
+                s.Grade = grade;
```

Comment:
s is not meaningful variable name. Please use variable names that are not abbreviations.